src/index.scss

```diff
@@ -1,10 +1,10 @@
-@import "_typography";
 @import "_colors";
 @import "_outline";
+@import "_typography";
```

Comment:
Any particular reason for this switcheroo?

Reply:
Oh maybe because it's alphabetical now haha

Reply:
Sorry, minor compulsion 😬

Reply:
I understand